src/en/news/blog/2025/rbd-live-migration/index.md

```diff
@@ -0,0 +1,478 @@
+---
+title: "Seamless Data Movement Across Clusters using RBD Live Migration in Squid"
+
+date: 2025-02-07
+author: Sunil Angadi, Anthony D'Atri (IBM)
+tags:
+  - ceph
+  - rbd
+---
+
+## Seamless Data Movement Across Clusters using RBD Live Migration in Squid
+
+### Introduction to Block Storage
+
+A block is a small, fixed-sized piece of data, like a 512-byte chunk. Imagine
+breaking a book into many pages: each page is a "block". When you put all the
+pages together, you get a complete book, just like combining many smaller data
+blocks creates a larger storage unit.
+
+#### Where Do We Use Blocks?
+
+Block-based storage is commonly used in computing, and in devices including:
+* Hard drives or SSDs (your computer's storage)
+* CDs/DVDs (discs you use for music or movies)
+* Floppy disks (older storage devices)
+* Tape drives (used in big data backup systems)
+
+#### What Are Ceph RADOS Block Devices (RBD)?
+
+Ceph Rados Block Devices are a type of storage served by a Ceph cluster that
```

Comment:
s/Rados/RADOS/

I think this is correct. Anyway, it appears that way in the title.

(It is correct to say "RADOS".)